projects/core/src/lib/shared/helpers/image.helper.ts

```diff
@@ -34,4 +35,26 @@ export class ImageHelper {
     }
     return subject.asObservable();
   }
+
+  public static string2Base64QRcode$(asciiData: string, foreground: string, background: string): Observable<{
+    imageData: string;
+    widthPx: number;
+    heightPx: number;
+  }> {
+    const encoder = new Encoder({
```

Comment:
Since this is an async method already maybe its nice to lazy load this qrcode module so it does not add to the bundle size when users never use this functionality. See for example the loading of jspdf (`from(import('jspdf'))`)